py/client/pydeephaven/session.py

```diff
@@ -249,6 +259,13 @@ def input_table_service(self) -> InputTableService:
 
         return self._input_table_service
 
+    @property
+    def plugin_object_service(self) -> PluginObjService:
```

Comment:
I _think_ this is not really meant to be user facing?

Reply:
If we don't want users to see it, can it be private?

Reply:
@jmao-denver

Reply:
It is not user facing but referenced by other modules, making it private will produce warnings from linters/static checkers etc. We have kinda adopted the convention that for such methods/classes etc., we don't provide a docstring so that Sphinx will not generate docs for them. Maybe we could just add comments.